src/PortToTripleSlash/src/libraries/ToTripleSlashPorter.cs

```diff
@@ -159,13 +159,11 @@ public async Task PortAsync(bool isMSBuildProject, CancellationToken cancellatio
             }
         }
 
-        private static void CollectSymbolLocations(Compilation compilation, string projectPath, DocsType docsType)
+        private static void CollectSymbolLocations(Compilation compilation, DocsType docsType)
         {
-            docsType.SymbolLocations = new List<ResolvedLocation>();
```

Comment:
This is already being initialized in the internal method. I added a null check a few lines below in this method.